source/_docs/guides/front-end-performance.md

```diff
@@ -0,0 +1,475 @@
+
+  <!-- 2nd Tab Nav -->
+  <li id="tab-2-id" role="presentation"><a href="#drupal-max-age" aria-controls="tab-2-anchor" role="tab" data-toggle="tab">Drupal</a></li>
+</ul>
+
+<!-- Tab panes -->
+<div class="tab-content">
+  <!-- Active pane content -->
+  <div role="tabpanel" class="tab-pane active" id="wordpress-max-age" markdown="1">
+  There could be an existing snippet assumed to disable caching for a single entity when in reality it's disabling caching for the entire page:
+
+  ```php
+  /**
+  * This is an example of accidentally bypassing cache when rendering a response.
+  * Don't use this without proper conditionals wrapped around the header function.
+  **/
+  add_action( 'send_headers', 'add_header_maxage' );
+  function add_header_maxage() {
+    // some logic that accidentally invalidates full-page cache
+      header('Cache Control: max-age=0');
+  }
+  ```
+   See the [WordPress documentation](https://codex.wordpress.org/Plugin_API/Action_Reference/send_headers){.external} for more details.
+  </div>
+
+  <!-- 2nd pane content -->
+  <div role="tabpanel" class="tab-pane" id="drupal-max-age" markdown="1">
+There could be an existing snippet assumed to disable caching for a single block when in reality it's disabling caching for the entire page:
+
+```php
+/**
+* This is an example of accidentally bypassing cache when rendering a response.
+* Don't use this without proper conditionals wrapped around it.
+**/
+// some logic that accidentally invalidates full-page cache
+$form['#cache'] = ['max-age' => 0];
+```
+ See the [Drupal documentation](https://www.drupal.org/docs/8/api/render-api/cacheability-of-render-arrays){.external} for more details. You can search your custom code for `#cache` to find places where you've interacted with the Cache API.
+  </div>
+</div>
+
+#### Lost or Forgotten Configuration
+Working across many environments in a distributed manner such as Pantheon presents opportunity for configuration to get lost along the way. Using tools like WP-CFM and Drupal 8's Configuration Management System to track configuration alongside code will mitigate these issues, but mistakes do happen. Double-check your site's default caching configurations:
+
+<!-- Nav tabs -->
+<ul class="nav nav-tabs" role="tablist">
+  <!-- Active tab -->
+  <li id="tab-1-id" role="presentation" class="active"><a href="#wp-config-get" aria-controls="wp-config-get" role="tab" data-toggle="tab">WordPress</a></li>
+
+  <!-- 2nd Tab Nav -->
+  <li id="tab-2-id" role="presentation"><a href="#drupal-config-get" aria-controls="drupal-config-get" role="tab" data-toggle="tab">Drupal</a></li>
+</ul>
+
+<!-- Tab panes -->
+<div class="tab-content">
+  <!-- Active pane content -->
+  <div role="tabpanel" class="tab-pane active" id="wp-config-get" markdown="1">
+  The Pantheon Page Cache plugin is already included by our upstream as a Must-Use plugin. Check the plugin settings to make sure you're setting the desired TTL and max-age:
```

Comment:
> the desired TTL and max-age

These are not different things, just different words for the same thing. Let's change it to "the desired TTL".